code/game/turfs/turf.dm

```diff
@@ -105,7 +105,7 @@
 					M.inertia_dir = 0
 			if(TURF_WET_LUBE)
 				M.slip(0, 4, null, (SLIDE|GALOSHES_DONT_HELP))
-				M.confused += 8
+				M.confused = 8
```

Comment:
M.confused = max(M.confused, 8) would be better, so that if they're super confused due to something else this doesn't lower it